### ISSUE TITLE
Fluent upgrade Accessibility testing by devs (automated accessibility insights tool)

### ISSUE THREAD

**Issue by ElinorW** [microsoftgraph/microsoft-graph-explorer-v4#3592]
- Run accessibility insights tool
- Check narrator
- Keyboard navigation
-
Issues detected 

- [x] main header buttons missing aria-labels 
- [x] signed out icon missing alt text
- [x] Resources version switch missing a label
- [x] missing labels on edit collection check box
- [x] missing tooltips on many icons e.g back arrow on collections, close button on dialogs, maximize/minimize sidebar
- [x] keyboard focus on the tab headers is cut off/not clear 

![Image](https://github.com/user-attachments/assets/fdff8df4-87da-470a-a104-145a08649e12)

![Image](https://github.com/user-attachments/assets/d8892f1c-7400-4da0-8e6a-8c4937e6084f)

![Image](https://github.com/user-attachments/assets/40c823bb-a4e1-4316-a3f9-2fc77952078a)

![Image](https://github.com/user-attachments/assets/00bd2e0a-5d0b-4108-a0d4-3e7328135c2e)

**Comment by Mnickii** [comment]
Closed by #3593
